scss/mixins/_media-queries.scss

```diff
@@ -57,31 +57,66 @@ $devices: (
   ),
 );
 
-@mixin breakpoint($size) {
-  @each $item, $value in $breakpoints {
-    @if $item == $size {
-      @if $value != null {
-        @media screen and #{$value} {
-          @content;
-        }
-      }
-    }
-  }
-}
 //@include device(device-name-here) {
 //  styles go here
 //}
 @mixin device($size) {
   @each $item, $value in $devices {
     @if $item == $size {
       @if $value != null {
-        @media only screen #{$value} {
+        @media #{only screen device-for($value)} {
```

Comment:
I wonder if this one should be `only screen #{device-for($value)}` like we changed in breakpoint

Reply:
it should, I got it